src/fides/api/models/privacy_notice.py

```diff
@@ -296,6 +296,15 @@ def cookies(self) -> List[Cookies]:
             )
         ).all()
 
+    @property
+    def systems_applicable(self) -> bool:
```

Comment:
I'm unsure about this—my instinct was to use a `property` or a `hybridproperty`, although I had trouble getting the `hybridproperty` to work. I was hoping once I had this field in place, I could use this field for the `systems_applicable` query param filter to simplify that logic a bit, but that didn't work out. I'm not sure if there's a way to make this property figure out `systems_applicable`, and then have the filter param `systems_applicable` just filter on this property, vs doing the logic again here in the endpoint? 

https://github.com/ethyca/fides/blob/a0544e57d42f58425b27e0b58b843e489ad1a671/src/fides/api/api/v1/endpoints/privacy_notice_endpoints.py#L58-L60

Reply:
I think this looks good to me as-is! From a code-sharing perspective, I can understand the motivation to want to have `systems_applicable` use that property to filter as well, but I think that would be less efficient than querying the `PrivacyNotice.data_uses` database column directly as we're doing now.   Granted, I can't imagine the PrivacyNotice table every being very big, so the practical difference between the two would be small.  But generally when querying an entire table, I try not to filter on a property but query the column directly to build the most straightforward and optimized sql query.

Reply:
ah got it, thanks for explaining!